ding/framework/middleware/functional/traffic_server.py

```diff
@@ -0,0 +1,46 @@
+from typing import TYPE_CHECKING, Callable, Union, List
+import time
+import logging
```

Comment:
DO NOT directly use `logging` package, replace `import logging` to `from ditk import logging` so all the log information are in control.

Reply:
Please also modify other relevant parts.

Reply:
fixed.